### DISCUSSION TITLE
[help]

### DISCUSSION THREAD

**Discussion by deanh2k** [Chia-Network/chia-blockchain#4616] [Support]
**Describe the issue**
Seeing these errors on port 8444, is port blocked on my end or target host?

How can I tell whether my system is functioning properly long term to actual earn Chia or forever stuck with hidden issues underneath?

2021-05-09T12:53:46.983 full_node full_node_server        : ERROR    Exception: Failed to fetch block 255587 from {'host': '91.12.162.3', 'port': 8444}, timed out <class 'ValueError'>, closing connection {'host': '91.12.162.3', 'port': 8444}. Traceback (most recent call last):
  File "chia\server\server.py", line 531, in api_call
  File "asyncio\tasks.py", line 414, in wait_for
  File "chia\server\server.py", line 528, in wrapped_coroutine
  File "chia\server\server.py", line 521, in wrapped_coroutine
  File "chia\full_node\full_node_api.py", line 104, in new_peak
  File "chia\full_node\full_node.py", line 361, in new_peak
  File "chia\full_node\full_node.py", line 310, in short_sync_backtrack
  File "chia\full_node\full_node.py", line 295, in short_sync_backtrack
ValueError: Failed to fetch block 255587 from {'host': '91.12.162.3', 'port': 8444}, timed out

**To Reproduce**
starting farming

**Expected behavior**
expect no errors for network

**Screenshots**
![image](https://user-images.githubusercontent.com/69939/117588639-7c9d2f80-b0d9-11eb-93f0-6a92db271dc2.png)

**Additional context**
run port scan to check

**Comment by Plopsi** [comment]
I have the same errors in the logs, fun-fact my errors are from 'None' :D - I think this has todo with the 1.1.4 to 1.1.5 update which seem to desync the network a bit until everyone has updated.

Just a couple of errors in the log, as far as I can tell, no problems otherwise. node is sync and farming.

**Comment by deanh2k** [comment]
thanks for boost in confidence...   since I have only 10 plots so far, I am not really sure if I am on track to earning any chia or just wishful thinking

**Comment by Plopsi** [comment]
In the end it's a matter of luck :D - whether you have the best proof for a block in your plots or not. For professional farming 1TiB is very low. But if you have the space, fill it and let it run. I currently filling my NAS, and will remove plots again when I need space. As the chia-blockchain-docker also runs on the NAS I will only keep the wallet on my main-pc and let the NAS do. MAybe switching step by step over to pool-mining when it get's properly available. So heads up :) Could be next block could be 10 years for a block with the right challenge so you have a proper proof for it.

**Comment by deanh2k** [comment]
I have QNAP with 10T of space, so you can run docker on it?

**Comment by Plopsi** [comment]
QNAP? Don't know, should be. I have a Synology - still have to use the SSH to pull the image and create the docker, as DSM doesn't support the ghcr.io repo in the interface. A little pain when updating, but I exported the settings and can realod them afterwards and start the container in the interface.

**Comment by BrandtH22** [comment]
If you are still experiencing the issues it is recommended to upgrade to the latest version of chia. Please download it from here: https://www.chia.net/downloads/

Since this issue has been open for a number of years without additional comments, we will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions, comments, or if the issue persists after an update.

```
Please note that upgrading major versions of chia generally requires the config.yaml to be recreated using these steps:
1. Stop chia
2. Rename the config or move it to a different directory
3. In CLI run chia init
4. Copy any custom settings from the old config to the new (plot drive directories, pool info, trusted node info, etc) - be careful here as yaml is very picky about formatting including those leading spaces
5. Start Chia
```

The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
